apps/settings/js/panels/simpin/simpin.js

```diff
@@ -126,15 +127,23 @@ define(function(require) {
       cardIndex = parseInt(cardIndex, 10);
 
       switch (type) {
-        case 'checkSimPin':
```

Comment:
In the previous PR, you asked:
> 

why not handle with this._elements.simPinContainer.addEventListener('change', this) ? Therefore we can keep this switch condition and remove simPinSwitched method.

The reason is due to the way that gaia-switch fires both click and change events. We can handle it inside of handle event, but then need to invert the clicked state, and that feels wrong to me. (Changed doesn't fire until after the click set is different). I think having a separate method is cleaner for now, I've added a comment.